### PR TITLE
Add kamel bind examples (1.9 branch)

### PR DESCRIPTION
**Pull request by claudio4j**
Add custom kamel bind commands to the examples
script/generator used this custom kamel bind for docs

### CODE REVIEW

**Comment by claudio4j** [comment]
@squakez for review and merge.

File: templates/bindings/camel-k/avro-deserialize-action-binding.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# dont_overwrite
+# kamel bind --name avro-deserialize-action-binding timer-source?message='{"first":"Ada","last":"Lovelace"}' --step json-deserialize-action --step avro-serialize-action -p step-1.schema='{"type": "record", "namespace": "com.example", "name": "FullName", "fields": [{"name": "first", "type": "string"},{"name": "last", "type": "string"}]}' --step avro-deserialize-action -p step-2.schema='{"type": "record", "namespace": "com.example", "name": "FullName", "fields": [{"name": "first", "type": "string"},{"name": "last", "type": "string"}]}' --step json-serialize-action log:info
```
Comment (squakez) [inline templates/bindings/camel-k/avro-deserialize-action-binding.yaml:1]: Are these comments meant to be here? I am not sure about that "dont_overwrite" comment before.

**Review by squakez — APPROVED** [review]
Looks good, thanks. Just a question though before merging.

File: templates/bindings/camel-k/avro-deserialize-action-binding.yaml (hunk 1 above)
Comment (claudio4j) [reply templates/bindings/camel-k/avro-deserialize-action-binding.yaml:1]: Yes, the comment is a marker to allow the generator.go to *not* generate the kamelet example file and the kamel bind example for the kamelet, and will source these examples into the .adoc files.

**Pull request merged**